index.js

```diff
@@ -72,9 +72,34 @@ var browserslist = function (selections, opts) {
         selections = selections.split(/,\s*/);
     }
 
+    var statsFile, statsExist;
+
     if ( opts.stats || process.env.BROWSERSLIST_STATS ) {
+        statsExist = true;
+    } else {
+        (function searchStatsFile(startPath) {
+            if (!fs.existsSync(startPath)) {
+                return;
+            }
+
+            var files = fs.readdirSync(startPath), filename, stat, currFile;
```

Comment:
Mixed variable difinition with values `var a = 1` and without `var a` is prohibited in this project :).